### DISCUSSION TITLE
How to change inpainting model?

### DISCUSSION THREAD

**Discussion by rayronvictor** [lllyasviel/Fooocus#1740] [Q&A] [answered]
I downloaded the https://civitai.com/models/4201?modelVersionId=245627 and put the model in the folder `models/inpaint`, but how to switch from the default model?

**Comment by xhoxye** [comment, marked as answer]
The model you mentioned is not an inpaint model, it is the base model for SD 1.5. The inpaint model of fooocus can only be selected from the original three versions of the model
